test/features/page_objects/poa.rb

```diff
@@ -0,0 +1,158 @@
+  end
+
+  def poa_requirements
+    @driver.find_element(:xpath, "//span[contains(.,'Shows your current address')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Matches the address you used on signup')]")
+    @driver.find_element(:xpath, "//span[contains(.,'Is your most recent document')]")
+  end
+
+  def start_verification_button
+    @driver.find_element(:xpath, "//button[contains(.,'Start verification')]")
+  end
+
+  ### Document selection screen elements
+  def document_selection_screen_title
+    @driver.find_element(:xpath, "//span[@class='onfido-sdk-ui-Title-titleSpan'][contains(.,'Select a UK document')]")
+  end
+
+  def document_selection_screen_subtitle
+    @driver.find_element(:xpath, "(//div[contains(.,'These are the documents most likely to show your current home address')])[9]")
+  end
+
+  def bank_building_statement_cell
+    @driver.find_element(:xpath, "//div[@class='onfido-sdk-ui-DocumentSelector-option'][contains(.,'Bank/Building Society Statemente-statements accepted')]")
+  end
+
+  def bank_building_statement_icon
+    @driver.find_element(:xpath, "//div[contains(@class,'onfido-sdk-ui-DocumentSelector-icon onfido-sdk-ui-DocumentSelector-icon-bank-building-society-statement')]")
+  end
+
+  def bank_building_statement_title
+    @driver.find_element(:xpath, "//p[@class='onfido-sdk-ui-DocumentSelector-label'][contains(.,'Bank/Building Society Statement')]")
+  end
+
+  def bank_building_statement_subtitle
+    @driver.find_element(:xpath, "(//div[@class='onfido-sdk-ui-DocumentSelector-tag'][contains(.,'e-statements accepted')])[1]")
+  end
+
+  def utility_bill_cell
+    @driver.find_element(:xpath, "//p[@class='onfido-sdk-ui-DocumentSelector-label'][contains(.,'Utility Bill')]")
+  end
+
+  def utility_bill_icon
+    @driver.find_element(:xpath, "//div[contains(@class,'onfido-sdk-ui-DocumentSelector-icon onfido-sdk-ui-DocumentSelector-icon-utility-bill')]")
+  end
+
+  def utility_bill_title
+    @driver.find_element(:xpath, "//p[@class='onfido-sdk-ui-DocumentSelector-label'][contains(.,'Utility Bill')]")
+  end
+
+  def utility_bill_subtitle
+    @driver.find_element(:xpath, "//div[@class='onfido-sdk-ui-DocumentSelector-hint'][contains(.,'Gas, electricity, water, landline, or broadband')]")
+    @driver.find_element(:xpath, "//div[@class='onfido-sdk-ui-DocumentSelector-warning'][contains(.,'Sorry, no mobile phone bills')]")
+    @driver.find_element(:xpath, "(//div[@class='onfido-sdk-ui-DocumentSelector-tag'][contains(.,'e-statements accepted')])[2]")
+  end
+
+  def council_tax_letter_cell
+    @driver.find_element(:xpath, "//p[@class='onfido-sdk-ui-DocumentSelector-label'][contains(.,'Council Tax Letter')]")
+  end
+
+  def council_tax_letter_icon
+    @driver.find_element(:xpath, "(//div[contains(@class,'onfido-sdk-ui-DocumentSelector-icon onfido-sdk-ui-DocumentSelector-icon-letter')])[1]")
+  end
+
+  def council_tax_letter_title
```

Comment:
This a duplicate

Reply:
This is still a duplicate :)

Reply:
Of what exactly? I cannot see the duplicate.

Reply:
Sorry, I meant that the body of the method was the same as `council_tax_letter_cell`. But `council_tax_letter_cell` body has changed so I was probably looking at the old code when I left the second message.